push-notifications/basics/ios.md

```diff
@@ -58,9 +69,65 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 }
 ```
 
+<a id="request-authorization"></a>
+## Request authorization to display notifications
+
+You should make sure your app has the permission to send user push notifications when the app is in the background according to the [Apple's documentation][apple-doc-registerforremotenotifications].
+
+::: caution
+
+If you do not request and receive authorization for your app's interactions, the system delivers all remote notifications to your app silently. It is adviced to ask the user for permission before registering Apple Push Notification Service .
+
+:::
+
+Here's how you can request for notification handler events:
+
+```obj-c
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+
+
+	UNUserNotificationCenter* center = [UNUserNotificationCenter currentNotificationCenter];
+	[center requestAuthorizationWithOptions:(UNAuthorizationOptionAlert + UNAuthorizationOptionSound)
+	   completionHandler:^(BOOL granted, NSError * _Nullable error) {
+	      // Enable or disable features based on authorization.
+	}];
+
+
+    [[[SKYContainer defaultContainer] push] registerDeviceCompletionHandler:^(NSString *deviceID, NSError *error) {
+        if (error) {
+            NSLog(@"Failed to register device: %@", error);
+            return;
+        }
+
+        // Anything you want to do in the callback can be added here
+    }];
+
+    // This will prompt the user for permission to send remote notification
+    [application registerForRemoteNotifications];
+
+    // Other application initialization logic here
+
+    return YES;
+}
+
+
+
```

Comment:
I found that the code is quite duplicate from the above section, excepts adding `requestAuthorizationWithOptions` call. Do you think we just update the first block of sample code?